server/avl/seqset.go

```diff
@@ -250,9 +276,19 @@ var (
 	ErrSetNotEmpty = errors.New("ss: set not empty")
 )
 
-func Decode(buf []byte) (*SequenceSet, error) {
-	if len(buf) < minLen || buf[0] != magic || buf[1] != version {
-		return nil, ErrBadEncoding
+// Decode returns the sequence set and number of bytes read from the buffer on success.
+func Decode(buf []byte) (*SequenceSet, int, error) {
```

Comment:
This is an exported function, so it is technically a breaking change for embedded usage. Low probability of being used, but noting..

Reply:
☝️ Ignore my comment.. this package (`avl`) is new for 2.10.. so nothing breaking here.

Reply:
It can decode both versions and I assume there will be these out there already so made sure that would work.

Reply:
I was referring to the signature changing, but since it's new to 2.10 it is not breaking.